src/modules/commands/config/viewSettings.ts

```diff
@@ -99,6 +99,7 @@ export const viewSettings = async (
       config.emote_channels.length.toString(),
       true
     );
+    settingsEmbed.addField("Ban Appeal Link", config.appeal_link, true);
```

Comment:
```suggestion
    settingsEmbed.addField("Ban Appeal Link", config.appeal_link || "not set", true);
```
Something like this... we need a fallback here for when there is no `appeal_link` value.
Committing this as-is might make the linter mad. Not sure.